### PR TITLE
Add mail2tor.com

### PR DESCRIPTION
**Pull request by link2xt**

### CODE REVIEW

File: _providers/mail2tor.md
Comment (missytake) [inline _providers/mail2tor.md:21]: That line isn't shown in the web interface: https://testrun.org/provider/ca6be5c3e6c2f87e35a8e891612dc8a0536201b7/_site/mail2tor

Maybe we should adjust _includes/provider.html?

**Review by missytake — APPROVED** [review]
Apart from that it's fine, and I don't know how important that line on the website really is.

**Comment by missytake** [comment]
Actually, installing tor is kind of a preparation step... maybe "OK" isn't really fitting?

File: _providers/mail2tor.md
Comment (missytake) [inline _providers/mail2tor.md:23]: Is there some information on configuration needed? Like "socks5 proxy at localhost:9050", or is it auto-configured if it sees a .onion domain?

**Review by missytake — APPROVED** [review]

File: _providers/mail2tor.md
Comment (link2xt) [reply _providers/mail2tor.md:23]: You need to configure socks proxy to localhost:9050, but afaik if you use Tor browser instead of the standalone Tor, then the port is different.

How to configure Tor is out of scope of the provider database, you can even use https://www.torbox.ch/ and avoid configuration of your client device completely.

Comment (missytake) [reply _providers/mail2tor.md:23]: Alright :) then let's merge it as it is, and at some point we can link to a blogpost which talks about configuration?

Comment (Jikstra) [reply _providers/mail2tor.md:23]: Is it possible to not list it on providers.delta.chat? It's mainly for conveniance of testing mail2tor addresses in my opinion, until socks5 support actually landed in a released version.

Comment (link2xt) [reply _providers/mail2tor.md:23]: It requires tweaking index.html and adding a new attribute like "hidden: true" to .yaml.

I don't think it's a problem for mail2tor that it's listed, it says clearly that you need to somehow use the Tor. If you can't, then you can't use the provider.

Comment (r10s) [reply _providers/mail2tor.md:23]: yip, i also think, listing is okay. however, for whatever reason, the preview seems to be broken and does not pick up the status `PREPARATION` and does not show the hints therefore. might be a preview-only issue, however.

File: _providers/mail2tor.md
Comment (r10s) [inline _providers/mail2tor.md:11]: the script `src/provider/update.py` crating `src/provider/data.rs` currently does not support `PLAIN`, i already have a fix for that, will do a pr.

Comment (r10s) [reply _providers/mail2tor.md:11]: pr: https://github.com/deltachat/deltachat-core-rust/pull/2604

File: _providers/mail2tor.md
Comment (missytake) [reply _providers/mail2tor.md:23]: The correct preview link is https://testrun.org/provider/2694122fdab5ac5094d212da3c7957ed9475f43a/_site/ - there it works.

The github actions create a new preview per commit instead of branch/PR... a bit weird behvaior, in most other repositories we solved this in a more clever way.

Just the comment providing the link is affected, if you access the preview from the checks below, it works - this one is updated.

File: _providers/mail2tor.md
Comment (missytake) [reply _providers/mail2tor.md:11]: as this one is merged... do we need to wait for a core release or can we merge this?

Comment (r10s) [reply _providers/mail2tor.md:11]: no need to wait, at least not for that reason :)

File: _providers/mail2tor.md
Comment (r10s) [reply _providers/mail2tor.md:23]: thanks for explanations. indeed, this is confusing. if it is not easily possible, to get a stable preview link, it would be better if the github-action post-bot says:

"Check out the page preview at: Show all checks / Page Preview / Details" 

instead of

"Check out the page preview at https://testrun.org/provider/1234567890...unstable-id/_site/"

Comment (r10s) [reply _providers/mail2tor.md:23]: i filed an issue for that at https://github.com/deltachat/provider-db/issues/197

**Pull request merged**
